stabilityai/pytorch/inference/buildspec.yml

```diff
@@ -9,40 +9,26 @@ arch_type: x86
 repository_info:
   inference_repository: &INFERENCE_REPOSITORY
     image_type: &INFERENCE_IMAGE_TYPE inference
-    root: !join [ *BASE_FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
+    root: !join [ "stabilityai/", *BASE_FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
     repository_name: &REPOSITORY_NAME !join [pr, "-", "stabilityai", "-", *BASE_FRAMEWORK, "-", *INFERENCE_IMAGE_TYPE]
     repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
 
 context:
   inference_context: &INFERENCE_CONTEXT
     torchserve-ec2-entrypoint:
-      source: docker/build_artifacts/torchserve-ec2-entrypoint.py
```

Comment:
If there are no additional artifacts, and we are building off of the base image, these are probably not necessary

Reply:
I got errors with them removed; planning to dig into why and resolve. I may add a custom entrypoint, however.

Reply:
I've removed for now - but we can add this back later if custom entrypoint is needed